### PR TITLE
Use Map for features

### PR DESCRIPTION
**Pull request by yordis**
Related to #282

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -114,7 +115,7 @@
     "#lint": "even if there is a .eslintignore symlink, we use an explicit command because windows don't like unix symlink",
     "lint": "eslint --ignore-path .gitignore .",
     "#tape": "to avoid really slow tests, we run babel once & run tests on the result",
-    "tape": "tape \"lib/__tests__/*.js\"",
+    "tape": "tape \"lib/__tests__/*.js\" | tap-spec",
```
Comment (yordis) [inline package.json]: @MoOx please support me on adding this. Was painful to read the logs of tape 😢

**Comment by yordis** [comment]
@MoOx another one down 🎉

**Comment by yordis** [comment]
hmm I have the Node version `v6.4.0` and it works -_-

**Comment by Semigradsky** [comment]
PR title: Use Map for features

PR content:
- switching features from object to Map
- changes in docs js files
- changing npm sript
- updating dependencies

?

**Comment by yordis** [comment]
@Semigradsky there is an issue with node 0.12.x 😞

**Comment by MoOx** [comment]
Closing in favor of #346

**Pull request closed without merging**